helm/kiam-app/charts/kiam-namespace-annotation/templates/np.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: {{ .Values.name }}
   namespace: {{ .Values.namespace }}
   annotations:
-    "helm.sh/hook": "pre-install"
+    "helm.sh/hook": "post-install"
```

Comment:
OK I get this now. The job is created with a post-install hook so these resources should also use post-install.

https://github.com/giantswarm/kiam-app/blob/98f6b166212332ee7bbb449616f73a162e0727fb/helm/kiam-app/charts/kiam-namespace-annotation/templates/job.yaml#L7

Reply:
Exactly!